docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.1'
+
+services:
+
+  postgres:
+    restart: always
+    image: postgres:11.4-alpine
+    expose:
+      - "5432"
+    environment:
+      - POSTGRES_USER=seven23
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=seven23
+
+  seven23_server:
+    build: .
+    image: seven23_server:latest
+    depends_on:
+     - postgres
+    ports:
+     - "8000:8000"
+    environment:
+      - DATABASE_URL=postgresql://seven23:password@postgres:5432/seven23
+      - CONTACT_EMAI=admin@seven23.io
+      - SECRET_KEY="MY-SECRET_KEY"
```

Comment:
```suggestion
      - SECRET_KEY=f01f979bcc9a6e3d8d10579da79998f120cad5310febe1027d59f3d306c9b0d0
```

Reply:
Django normally **generate** such key when running `startproject` (see [documentation about Cryptographic signing](https://docs.djangoproject.com/en/2.2/topics/signing/)).

I know myself and would be too lazy to change it, which might not be great in terms of security 😅. I think we need to generate one on build.

Reply:
If I am not mistaken, defining it in the docker-compose.yml will only make it usable in development. Which seems to be fine.

If people use the docker image in production they would change it when generating their kubernetes config I guess.